state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -122,6 +122,9 @@ pub mod pallet {
 	#[pallet::config]
 	#[pallet::disable_frame_system_supertrait_check]
 	pub trait Config<I: 'static = ()>: Chainflip {
+		/// Name of the chain that this Config is implemented for
+		const NAME: &'static str;
```

Comment:
Can we constrain this to match the TargetChain::NAME?

(Although I think simply creating a trait or passing the name in as a derive arg would be nicer, for example 
```rust
#[derive(GenericTypeInfo)]
#[generic_type_info("<T as TargetChain>::NAME")]
```
Then we don't need to make any assumptions on what the traits contain.
)